src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 100;
+    private static final int defaultRadius = 10;
+
+    public Figure getRandomFigure() {
+        Rectangle randomRectangle = new Rectangle(new Random().nextInt(BOUND),
+                new Random().nextInt(BOUND));
+        randomRectangle.setColor(new ColorSupplier().getRandomColor());
+
+        Square randomSquare = new Square(new Random().nextInt(BOUND));
+        randomSquare.setColor(new ColorSupplier().getRandomColor());
+
+        Circle randomCircle = new Circle(new Random().nextInt(BOUND));
+        randomCircle.setColor(new ColorSupplier().getRandomColor());
+
+        IsoscelesTrapezoid randomIsoscelesTrapezoid = new IsoscelesTrapezoid(new Random()
+                .nextInt(BOUND), new Random().nextInt(BOUND),new Random().nextInt(BOUND));
+        randomIsoscelesTrapezoid.setColor(new ColorSupplier().getRandomColor());
+
+        RightTriangle randomRightTriangle = new RightTriangle(new Random().nextInt(BOUND),
+                new Random().nextInt(BOUND));
+        randomRightTriangle.setColor(new ColorSupplier().getRandomColor());
+
+        String randomFigure = Figures.values()[new Random().nextInt(Figures.values().length)]
+                .name();
+
+        switch (randomFigure) {
+            case "CIRCLE" :
+                return randomCircle;
```

Comment:
```suggestion
        int randomFigure = random.nextInt(FIGURE_NUMBER);

        switch (randomFigure) {
            case 0 :
                return getRandomCircle();
```
lets use just numbers to choose which figure we should create
also lets make separate private methods in this class to get random figures